foundry.toml

```diff
@@ -4,10 +4,9 @@
   auto_detect_solc = false
   bytecode_hash = "none"
   cbor_metadata = false
-  gas_reports = ["*"]
-  ignored_error_codes = [
-    5159, # ignore selfdestruct warning
-  ]
+  fs_permissions = [{ access = "read", path = "./optimized-out" }]
+  gas_reports = ["PRBProxy", "PRBProxyFactory", "PRBProxyRegistry"]
+  ignored_error_codes = [5159]
```

Comment:
These changes should not have been introduced in this PR (they have been made on `staging`).

Will remove them in a commit, and then I will merge.